distributed/deploy/tests/test_cluster.py

```diff
@@ -33,14 +33,3 @@ async def test_logs_deprecated():
     cluster = Cluster(asynchronous=True)
     with pytest.warns(FutureWarning, match="get_logs"):
         cluster.logs()
-
-
-@gen_test()
-async def test_cluster_info():
-    class FooCluster(Cluster):
-        def __init__(self):
-            self._cluster_info["foo"] = "bar"
-            super().__init__(asynchronous=False)
-
-    cluster = FooCluster()
```

Comment:
Why are we removing this test? We want to depend on this functionality downstream so should be testing for it.

This change is technically a break in terms of API because we will need to wait until after the `super().__init__()` has been called before we can mutate the `_cluster_info` dict. But that should be totally fine. 

Let's just update this test to reflect that.

Reply:
Sure. Ideally the test would asset that the state is successfully passed to the scheduler and can be retrieved. 

But removing the test all together is moving in the wrong direction.